src/PhpunitMerger/Command/LogCommand.php

```diff
@@ -82,7 +82,8 @@ private function addTestSuites(\DOMElement $parent, array $testSuites)
         foreach ($testSuites as $testSuite) {
             if (empty($testSuite['@attributes']['name'])) {
                 if (!empty($testSuite['testsuite'])) {
-                    $this->addTestSuites($parent, $testSuite['testsuite']);
```

Comment:
Hi. Thank for contributing. I am trying also to use this package.
Can we add some tests for the case that you added in order to reproduce the issue then the fix?